app/bounty_requests/templates/bounty_request_form.html

```diff
@@ -29,47 +29,51 @@
     <div class="row no-gutter">
       <div class="col body">
         <div class="container-fluid no-gutter">
-          <div class="m-4">
-            <div>
-              <div class="request-form" id="primary_form">
-                <h3>{% trans "Request a Bounty" %} <span id="alpha">{% trans "Alpha" %}</span></h3>
-                <p>
-                  {% blocktrans %}
-                  Have an open-source issue that you think would benefit the community?
-                  Suggest it be given a bounty!
-                  {% endblocktrans %}
-                </p>
-                <div class="mb-3">
-                  <label for="requested_by" class="form__label">{% trans 'Your Github Profile' %}
-                    {% if not user.is_authenticated %}
-                    (<a href="{% url 'social:begin' 'github' %}?next={{ request.get_full_path }}">{% trans 'Login' %}</a>)
-                    {% endif %}
-                  </label>
-                  <input name="requested_by" placeholder="{% trans "Please Login First" %}" id="requested_by" class="form__input" type="text" value="{% if github_handle %}{{ github_handle }}{% endif %}" required disabled />
-                </div>
-                <form id="bounty_request_form">
-                  {% for field in form %}
-                    <div class="mb-3 {{ field.name }}">
-                      <label for="id_{{ field.name }}" class="form__label">{{ field.label }}</label>
-                      {{ field }}
+          <div class="row">
```

Comment:
not clear on why we need another row when we just defined it above.
I'm pretty sure this can be done in a neater way